AdminCore.Services/DashboardService.cs

```diff
@@ -84,14 +83,14 @@ public IList<ClientSnapshotDto> GetTeamDashboardEvents(int employeeId, DateTime
       var teamsForEmployee = GetTeamIdsForEmployee(employeeId, date);
       var clientList = DatabaseContext.ClientRepository
         .GetAsQueryable(QueryClientsWithContractsForEmployeeId(teamsForEmployee, date))
-        .Include(client => client.Projects)
+        ?.Include(client => client.Projects)
```

Comment:
I don't think we need optional unwrapping here.

Reply:
Removed